src/index.js

```diff
@@ -129,7 +129,7 @@ class Anvil {
 
     return {
       statusCode,
-      url: data?.data?.generateEtchSignURL,
+      url: data && data.data && data.data.generateEtchSignURL,
```

Comment:
I think we need to discuss a bit more what versions of Node we want to support in this package, and how we want to manage that. I don't think we're specifying what versions we support, and when I dropped down to node 8, this code here would not work.

Reply:
Yeah, I think node 8 is a reasonable assumption

Reply:
well thumb me up here then unless you've got other comments... 😄